README.md

```diff
@@ -5,3 +5,11 @@ To set up a virtual environment for development, first [install pypoetry onto yo
 From there, you can [install your project dependencies](https://python-poetry.org/docs/basic-usage/#installing-dependencies) and
 [run any scripts in the library](https://python-poetry.org/docs/basic-usage/#using-poetry-run) within the virtualenv. You can also
 [build the contents of the repo into a python .whl](https://python-poetry.org/docs/cli#build) which can then be installed anywhere via `pip`
+
+
+### Manual Override
+Sometimes, the robots fail us and we have to take PDH into our own hands. Quick Commander_json manual update Guide.
+
+1) Make sure [Poetry](https://python-poetry.org/docs/) is installed on your local machine, this is necessary to run the scripts
+2) run ```poetry install```, if you're on mac/windows you'll need ```poetry install --no-dev``` since dev dependencies aren't supported
```

Comment:
Specifically the package that wasn't supported was `lazy-object-proxy`, not sure if we want to make note of that. Also I don't know for sure if its not supported on windows, but regardless `poetry install --no-dev` should always work

Reply:
I think their site says windows issues but I'll double check and add specifics for the failing packages